src/sdk/comment.ts

```diff
@@ -0,0 +1,48 @@
+import { Context } from "./context";
+import { LogReturn } from "@ubiquity-os/ubiquity-os-logger";
+import { sanitizeMetadata } from "./util";
+
+const HEADER_NAME = "Ubiquity";
+
+/**
+ * Posts a comment on a GitHub issue if the issue exists in the context payload, embedding structured metadata to it.
+ */
+export async function postComment(context: Context, message: LogReturn) {
+  if ("issue" in context.payload && context.payload.repository?.owner?.login) {
```

Comment:
Should we add `if("pull_request" in ...)` as well and use `pull_request.number`?

I'm not 100% if payloads with `"pull_request"` in it always have `"issue"` too